pta-intelligent-search-api/src/test/resources/fi/maanmittauslaitos/pta/search/api/region/test_municipalities.json

```diff
@@ -55,9 +55,15 @@
       "properties": {
         "nimi": "Helsinki",
         "namn": "Helsingfors",
-        "synonyms_fi": ["Stadi"],
-        "synonyms_sv": ["Stad"],
-        "synonyms_en": ["City"],
+        "synonyms_fi": [
+          "Stadi"
+        ],
+        "synonyms_sv": [
+          "Stad"
+        ],
+        "synonyms_en": [
+          "City"
```

Comment:
Same as above. "City of Tampere" would/could map to Helsinki

Reply:
This is just for testing purposes, but I'll remove these as well.